### DISCUSSION TITLE
Is there any way to adjust nesting level of a list item from mobile device?

### DISCUSSION THREAD

**Discussion by PoisonousJohn** [outline/outline#7594] [Self Hosting] [answered]
On desktop browser I use tab/shift+tab, but how to do that on mobile version?

**Comment by tommoor** [comment]
![image](https://github.com/user-attachments/assets/52fd27cc-f8a1-499f-89a6-61ae4e71ca3d)

The icons on the right of the toolbar offer indent and outdent on mobile.

**Reply by PoisonousJohn** [reply]
I don't have that bar on mobile browser (Chrome Android)

**Reply by tommoor** [reply]
Please post a screenshot, perhaps it's hidden behind the keyboard?

**Reply by PoisonousJohn** [reply]
![telegram-cloud-photo-size-2-5413396653148333075-y](https://github.com/user-attachments/assets/0e94a819-20d5-452e-9083-6d1a61cb9d69)

**Reply by tommoor** [reply, marked as answer]
Are you using the cloud version or self-hosted? The toolbar does get hidden behind the keyboard on Android in older versions

**Reply by PoisonousJohn** [reply]
I use self-hosted version. Thanks, fixed by updating to 0.79.1
